### ISSUE TITLE
[CM13.0] [iyokan] 'Clicking' noise instead of silence if using PC as audio device

### ISSUE THREAD

**Issue by avsaponchik** [LegacyXperia/local_manifests#1022]
1. Iyokan
2. cm12.1 latest public build
3. 13.0-20160203 build from Adrian DC
4. I have no problems while using my headphones + iyokan
5. This problem only persists while no sounds/music/notifications/etc being played on my iyokan
   ![s-l225 1](https://cloud.githubusercontent.com/assets/1337953/12833292/c01bfb74-cbb0-11e5-8ebd-af695f272c3e.jpg)

Sorry for my russian

**Comment by s0rl0v** [comment]
Hi. Is the noise present, if you are not using the device?

**Comment by bykidi** [comment]
Few days ago it was clicking by itself every 0.8-1 second if i dont using it, now it only clicks once. Dno what i've done to 'fix' it.

**Comment by s0rl0v** [comment]
We'll see. I will try to reproduce your case on my urushi.

Can you check if the problem can be reproduced on a different PC?

**Comment by s0rl0v** [comment]
My device also has this issue :)

**Comment by bykidi** [comment]
problem is back again, even bigger now http://bykidi.tk/audioclicking2.mp3

**Comment by avsaponchik** [comment]
tested with 13.0, problem is still here

**Comment by s0rl0v** [comment]
All works well with headphones, right? We need to check if software is the culprit.

Can someone try to reproduce this problem on stock ROM?

**Comment by realcatgirly** [comment]
The device has to check if there is a headphone connected or not. Maybe it sends these "clicks" through the jack to detect that. On a regular earphone you cannot hear it but maybe the PC amplifies that and plays that via the speakers.
I can hear this clicks when I use my Bluetooth speaker in aux mode. In Bluetooth mode there is nothing.

**Comment by avsaponchik** [comment]
I got my hands on Neo V today, flashed it to latest LX13, same issue persists

**Comment by s0rl0v** [comment]
I connected my urushi to my car's aux input with high impendance. All good, noise is gone.
It smells like a hardware problem :smile:

**Comment by MardonHH** [comment]
was the device plugged into charger or usb while playing audio on pc?

**Comment by avsaponchik** [comment]
No. It makes more noise if you plug USB to computer, but if you power it up with charger there is less noise.
Best result you can get is without using any external power supply. But this bug is just clicking while no music being played. At all.

**Comment by rtammekivi** [comment]
Do you have 4 pole audio cable or 3 pole? Might be worth to swap it. Or maybe separate the microphone pole on the other side. I think its some part of the hardware that detects cable or similar.

I noticed same when trying my Huawei msm7x30 tv out functionality. Problem is almost all of that code is in the modem and not modifiable.

**Comment by avsaponchik** [comment]
@Blefish 
3-pole
Fresh build, iyokan, problem is still here
